src/components/TokenLocking/ActivityRewardsInfo.tsx

```diff
@@ -72,77 +67,33 @@ export const ActivityRewardsInfo = () => {
             <Typography variant="h1" fontWeight={700} mt="52px" mb="40px" maxWidth="60%" className={css.gradientText}>
               Interact with Safe and get rewards
             </Typography>
-
             <Step
               active={stepsActive[0]}
-              title="Lock SAFE to boost your miles!"
-              description="Lock your SAFE tokens early and increase your mile earning power. The earlier and more you lock, the bigger
-          your miles multiplier."
+              title="Lock SAFE to boost Points"
+              description="Lock your tokens early to increase your earning power. The earlier and more you lock, the bigger your points multiplier."
             />
-
             <Step
               active={stepsActive[1]}
-              activationDate="June 10"
-              title={
-                <>
-                  Get activity miles
-                  {!stepsActive[1] && (
-                    <Typography variant="caption" component="span" className={css.comingSoon}>
-                      Coming soon
-                    </Typography>
-                  )}
-                </>
-              }
-              description="Earn miles by using your Safe Account. Your miles are multiplied by the boost you build until September 30."
+              title="Get activity Points"
+              description="Earn Points by using your Safe Account. Your Points are multiplied by the boost you build until end of Season 1."
```

Comment:
> Your Points are multiplied by the boost you build until end of Season

As far as I know this is not true anymore. The realized boost is applied to all gained points. But we can adjust it again so not a blocker for this PR.